### DISCUSSION TITLE
Is it possible to disable some of the features?

### DISCUSSION THREAD

**Discussion by osrl** [gonzalezreal/swift-markdown-ui#345] [Q&A]
For example, I want to disable headings, blocks, thematic breaks etc.

**Comment by TimCadesso** [comment]
This is something I also would like to know. Currently I am overwriting the styles of this Blocks & Labels

**Comment by TimCadesso** [comment]
OK, looks like you can do it at least with .renderPlainText() for MarkdownContent

Like here:
```
.blockquote { configuration in
                Text(configuration.content.renderPlainText())
            }
```
But a initial setup config with [.table, .code, ...] als allowed or disallowed elements would also be nice. Maybe with the option keep syntax characters or not.
